zetaclient/zetabridge/zetacore_bridge.go

```diff
@@ -220,14 +220,15 @@ func (b *ZetaCoreBridge) UpdateZetaCoreContext(coreContext *corecontext.ZetaCore
 	var newBTCParams *observertypes.ChainParams
 
 	// check and update chain params for each chain
+	sampledLogger := b.logger.Sample(&zerolog.BasicSampler{N: 10})
```

Comment:
Is this used to reduce the frequency of the output logs?

Reply:
Yeah.